website/docs/language/settings/backends/azurerm.mdx

```diff
@@ -249,6 +249,8 @@ The following configuration options are supported:
 
 ~> **NOTE:** An `endpoint` should only be configured when using Azure Stack.
 
+* 'metadata_host' - (Optional) The Hostname of the Azure Metadata Service (for example management.azure.com), used to obtain the Cloud Environment when using a Custom Azure Environment. This can also be sourced from the ARM_METADATA_HOSTNAME Environment Variable.
```

Comment:
```suggestion
* `metadata_host` - (Optional) The Hostname of the Azure Metadata Service (for example `management.azure.com`), used to obtain the Cloud Environment when using a Custom Azure Environment. This can also be sourced from the `ARM_METADATA_HOSTNAME` Environment Variable.
```